### ISSUE TITLE
Opera

### ISSUE THREAD

**Issue by susceptive** [JourneyOver/Youtube-Video-Blocker#5]
Not work on Opera 47.

**Comment by JourneyOver** [comment]
Yes I know it's not working on the latest youtube design, I don't really have an estimate time on when I will be able to work on this..
